lib/chef/cookbook/metadata.rb

```diff
@@ -618,12 +618,11 @@ def self.from_json(string)
 
       def self.validate_json(json_str)
```

Comment:
per other comments below this should be `def validate_json(json_str)` -- however you should keep this API intact and still have a class method that builds an instance and validates it so that you don't break any other consumers of it (consider the class method a public API that you can't drop).